backend/tests/conftest.py

```
@@ -0,0 +1,274 @@
+"""
+
+import pytest
+import os
+import sys
+from datetime import datetime
+
+# Add the backend directory to the Python path
+sys.path.insert(0, os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+from app import create_app
+from models import db, User, Tool, Chemical, UserActivity, AuditLog
```

Ruff (0.11.9)

`models.AuditLog` imported but unused

Remove unused import: `models.AuditLog`

(F401)